test/problematic_tests/impl_test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import pytest
+import unittest
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+from conftest import assert_exception_correct
+
+
+class ImplTest(unittest.TestCase):
+    def setUp(self):
+        self.cfg = {}
+        self.cfg["workspace-url"] = "https://ci.kbase.us/services/ws"
+        self.cfg["shock-url"] = "https://ci.kbase.us/services/shock-api"
+        self.cfg["handle-service-url"] = "https://ci.kbase.us/services/handle_service"
+        self.cfg["SDK_CALLBACK_URL"] = "http://ip:port"
```

Comment:
```suggestion
```

Reply:
`SDK_CALLBACK_URL` env was set when initiate `kb-sdk test`?

Reply:
But it doesn't wind up in the dict passed to the constructor

Reply:
:+1: